### PR TITLE
Added a section for common iOS documents

### PR DESCRIPTION
**Pull request by NSElvis**
I guess this belongs to Hyper and should live here, in case the test gets updated or changed please update this document.

I remember looking at [Square](http://corner.squareup.com/) and their open test, it was a real challenge, it was so simple, so tangible, so doable, but challenging, because you had all the time in the world to do the test, it was there, the description was there, you just had to do it, do it great, do it exceptional and ship it to them.

This is the idea :)

### CODE REVIEW

File: iOS/HYPER_RECIPES.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Hi,
+
+Thanks for taking some time to talk with us, as we told you after the interview we usually have a fun small project called Hyper Recipes.
+
+The goal is to make an iOS app to see recipes it can be a list or a grid, just the listing part will do the job but you have other endpoints available in case youâ€™re feeling adventurous. You decide how it looks, but we appreciate good looking apps and thoughtful user experiences.
```
Comment (timkurvers) [inline iOS/HYPER_RECIPES.md]: Are we using some form of special quotes / apostrophes here?

Comment (NSElvis) [inline iOS/HYPER_RECIPES.md]: @timkurvers Weird, anyway I fixed the thing :)

**Comment by zenangst** [comment]
I like this!

**Comment by NSElvis** [comment]
@zenangst You can follow up your like with a merge!

**Comment by jgorset** [comment]
:+1:

**Pull request merged**